CONTRIBUTING.rst

```diff
@@ -0,0 +1,67 @@
+How to contribute
+=================
+
+Welcome to *coala*! The following guidelines will give you a short overview 
+over how we handle issues and PRs in this repository. *coala* is maintained
+by a growing community and newcomers are always welcome!
+
+We appreciate all valuable contributions - bug reports, documentation and
+design enhancements issues as well as pull Requests to make *coala* more
+convenient, functional and bug free.
+
+To begin contributing have a look at the open issues labeled with
+difficulty/newcomer - `newcomer issues`_.
```

Comment:
no, the links were lost. It should be `newcomer issues <https://github.com/coala-analyzer/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`__ instead of `newcomer issues`_